### PR TITLE
Delete administrator

### PR DESCRIPTION
**Pull request by seanpdoyle**
screen shot 2016-04-19 at 3 12 05 pm

https://trello.com/c/7NfQvJ6U

Enables the `DELETE /users/:id` action to remove a user from the system.

### CODE REVIEW

**Comment by corwinharrell** [comment]
✅

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -2,4 +2,12 @@ class UsersController < ApplicationController
   def index
     @users = User.all
   end
+
+  def destroy
+    @user = User.find(params[:id])
```
Comment (kenyonj) [inline app/controllers/users_controller.rb]: Thoughts on removing the ivar here?

**Comment by corwinharrell** [comment]
@seanpdoyle made some small changes to get those table links looking right:

screen shot 2016-04-19 at 1 36 30 pm

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (seanpdoyle) [inline app/controllers/users_controller.rb]: @kenyonj done.

File: app/views/users/_admins.html.erb [hunk 2]
```diff
@@ -9,10 +9,12 @@
   </thead>
   <tbody>
     <% admins.each do |admin| %>
-      <tr>
+      <%= content_tag_for(:tr, admin) do %>
         <td><%= admin.name %></td>
-        <td class="table-actions"><a href="" class="table-link table-delete">Delete</a></td>
-      </tr>
+        <td class="table-actions">
+          <%= render("users/delete", user: admin) %>
```
Comment (kenyonj) [inline app/views/users/_admins.html.erb]: Do you need to specify `users` here since you are already in that context?

Comment (seanpdoyle) [inline app/views/users/_admins.html.erb]: I've decided to be deliberately explicit about partial names in this project -- it's frustrating when refactoring/renaming a parent partial breaks children partials.

**Pull request merged**